roles/ceph-facts/tasks/devices.yml

```diff
@@ -85,4 +87,5 @@
     - item.value.sectors != "0"
     - item.value.partitions|count == 0
     - item.value.holders|count == 0
+    - ansible_facts['mounts'] | json_query('[?device==device]') | length == 0
```

Comment:
@clwluvw the filter `json_query` requires `jmespath`
currently we don't have this dependence in our `requirements.tx` (only in `tests/requirements.txt` given that we were using `json_query` in tests only)

Reply:
@guits Done. PTAL